heroku_connect/db/models/base.py

```diff
@@ -1,12 +1,26 @@
 from django.core import checks
-from django.db import models
+from django.db import NotSupportedError, models
 
 from . import fields
 from ...conf import settings
 
 __all__ = ('HerokuConnectModel',)
 
 
+class HerokuConnectModelManager(models.Manager):
```

Comment:
You should overwrite the `Queryset` and create a manager as `Queryset.as_manager()`.